app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :jwt_authenticatable, jwt_revocation_strategy: JwtBlacklist
+
+  has_many :reservations, dependent: :destroy
+  has_many :cars, through: :reservations
+
+  validates :name, :email, presence: true
+
+  def admin?
+    role == 'admin'
```

Comment:
[OPTIONAL]
- Consider adding a default role
- It is better to use enum for role to avoid unnecessary errors